src/axom/spin/internal/linear_bvh/build_radix_tree.hpp

```diff
@@ -528,6 +532,25 @@ AXOM_HOST_DEVICE static inline void sync_store(BBoxType& box,
 #endif
 }
 
+//------------------------------------------------------------------------------
```

Comment:
Code above here produces a warning when I compile since macros aren't defined: "build_radix_tree.hpp(454): warning: variable "nreads" was declared but never referenced"

Reply:
That warning should be suppressed now.